src/components/SelectionList/BaseListItem.tsx

```diff
@@ -25,13 +24,12 @@ function BaseListItem({
     onSelectRow,
     onDismissError = () => {},
     keyForList,
-}) {
+}: BaseListItemProps) {
     const theme = useTheme();
     const styles = useThemeStyles();
     const StyleUtils = useStyleUtils();
     const {translate} = useLocalize();
-    const isUserItem = lodashGet(item, 'icons.length', 0) > 0;
-    const ListItem = isUserItem ? UserListItem : RadioListItem;
+    const isRadioItem = item.rightElement === undefined;
```

Comment:
why this is different from original?

Reply:
It is simplier and easier to differ types `User` and `RadioItem` as `rightElement` is required for `User` type